app/src/main/java/com/infomaniak/mail/MatomoMail.kt

```diff
@@ -116,6 +131,21 @@ object MatomoMail : MatomoCore {
         trackEvent("createFolder", name)
     }
 
+    fun Context.trackMultiSelectionEvent(name: String, action: TrackerAction = TrackerAction.CLICK) {
+        trackEvent("multiSelection", name, action)
+    }
+
+    fun Fragment.trackMultiSelectActionEvent(name: String, value: Int, isFromBottomSheet: Boolean = false) {
+        val trackerName = "${if (value == 1) "bulkSingle" else "bulk"}${name.capitalizeFirstChar()}"
+
+        if (isFromBottomSheet) {
+            trackBottomSheetMultiSelectThreadActionsEvent(trackerName, value)
+        } else {
+            trackMultiSelectThreadActionsEvent(trackerName, value)
+        }
+    }
+
     // We need to invert this logical value to keep a coherent value for analytics
+    // because mail action conditions are inverted (ex: if the condition is message.isSpam, then we want to unspam the message)
```

Comment:
```suggestion
    // We need to invert this logical value to keep a coherent value for analytics because Messages actions
    // conditions are inverted (ex: if the condition is `message.isSpam`, then we want to unspam the Message)
```